distributed/diagnostics/websocket.py

```diff
@@ -0,0 +1,54 @@
+import tornado.websocket
+
+from .plugin import SchedulerPlugin
+from ..utils import key_split
+from .task_stream import colors
+
+
+class WebsocketPlugin(SchedulerPlugin):
+    def __init__(self, socket: tornado.websocket.WebSocketHandler, scheduler):
```

Comment:
I think that the type here is incorrect.  I think that this is an `EventsHandler`

Reply:
Good point! Not sure why I decided specify the base class instead.